packages/fabric8-ui/src/app/space/add-app-overlay/add-app-overlay.component.ts

```diff
@@ -23,6 +23,10 @@ import {
   DeploymentApiService,
 } from '../create/deployments/services/deployment-api.service';
 
+export type appModal = {
+  show: boolean;
+  selectedFlow: string;
+};
```

Comment:
Why not put a space after this? I guess it can get fixed by prettier or the linter.